lib/open_api/serializer.rb

```diff
@@ -8,9 +8,12 @@ def as_json(arg = {})
       version = [major, minor].compact.join(".")
       schema  = Api::Docs[version].definitions[self.class.name]
       attrs   = encryption_filtered.slice(*schema["properties"].keys)
+      encrypted_columns_set = (self.class.try(:encrypted_columns) || []).to_set
```

Comment:
Could you do this up [here](https://github.com/ManageIQ/topological_inventory-api/blob/cfef5d522435656287ebe3b8389606660508ea89/lib/open_api/serializer.rb#L5) and reuse the encrypted_columns_set for the encryption_filtered?

Reply:
done